content/chainguard/chainguard-images/how-to-use/use-with-openshift.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "How to Use Chainguard Containers with OpenShift"
+linktitle: "Using With OpenShift"
+type: "article"
+description: "Special considerations for using Chainguard Containers with OpenShift"
+date: 2025-06-17T08:49:31+00:00
+lastmod: 2025-06-17T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "OpenShift", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 008
+toc: true
+---
+
+When [Using Chainguard Containers](./how-to-use-chainguard-images.md) with OpenShift, there are some adjustments that need to be made to the usual process.
```

Comment:
Can you add more of an introduction, that addresses why to use Chainguard Containers and OpenShift? High level of why do we care as long as some potential use cases (use cases I would just list in a sentence as long as it's concrete enough)